### PR TITLE
updates to restaurantlist, about page, responsiveness for error and n…

### PR DESCRIPTION
**Pull request by Khemmie-Ray**
…oresults

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
   },
   "dependencies": {
     "axios": "^1.3.5",
+    "dotenv": "^16.0.3",
```
Comment (cherylli) [inline package.json:13]: we don't need dotenv for this project

File: src/pages/RestaurantList/RestaurantList.jsx [hunk 2]
```diff
@@ -1,36 +1,43 @@
...
 import Modal from "../../components/RestaurantCard/Modal";
 import { useState } from "react";
 
-const RestaurantList = () => {
+const RestaurantList = (props) => {
   const [show, setShow] = useState(false);
+  const [selectedData, setSelectedData] = useState(null)
+  const location = useLocation()
+  const propsData = location.state
+  // console.log(propsData.mapData)
```
Comment (cherylli) [inline src/pages/RestaurantList/RestaurantList.jsx:12]: this can be removed

**Review by cherylli — CHANGES_REQUESTED** [review]
Just a couple of things otherwise works fine

File: package.json (hunk 1 above)
Comment (Khemmie-Ray) [reply package.json:13]: I will take it out I forgot I installed it when I had the .env issue

File: src/pages/RestaurantList/RestaurantList.jsx (hunk 2 above)
Comment (Khemmie-Ray) [reply src/pages/RestaurantList/RestaurantList.jsx:12]: You mean the console.log?

**Review by cherylli — APPROVED** [review]
Thank you

**Pull request merged**